torch/csrc/Module.cpp

```diff
@@ -601,6 +614,14 @@ PyObject *THPModule_setFlushDenormal(PyObject *_unused, PyObject *arg) {
   Py_RETURN_TRUE;
 }
 
+static PyObject* THPModule_initializeTensorTypes(PyObject *_unused)
```

Comment:
why not just call this initializePythonBindings?

Reply:
We have lots of functions that initialize Python bindings of some sort. Open to suggestions for other names, though

Reply:
You are right, I'd be fine with initializeTensorTypeBindings, but if you want to leave it that's okay too.